daliuge-translator/dlg/dropmake/web/translator_rest.py

```diff
@@ -635,7 +635,7 @@ def gen_pg(
             host=mhost, port=mport, url_prefix=mprefix, timeout=30
         )
         # 1. get a list of nodes
-        node_list = mgr_client.nodes()
+        node_list = [f"{mhost}:{mport}"] + mgr_client.nodes()
```

Comment:
@awicenec Are you happy with this approach to the fix? I recall you saying in-person you'd made changes to the API instead, but that may have been orthogonal to this issue.

Reply:
These changes are perfect, thanks.